helm/prometheus-rules/templates/recording-rules/service-level.rules.yml

```diff
@@ -38,6 +38,45 @@ spec:
         service: api-server
       record: slo_target
 
+      # -- daemonset
+    - expr: |
+        label_replace(
+          kube_daemonset_status_desired_number_scheduled{cluster_id!="argali|giraffe", namespace=~"giantswarm|kube-system", daemonset=~"aws-node|kiam-server|kiam-agent"},
+        "service", "$1", "workload_name", "(.*)" )
+      labels:
+        class: MEDIUM
+        area: kaas
+      record: raw_slo_requests
+      # -- the errors are counted as follows: 
+      # -- pods in a daemonset that are UNAVAILABLE NOW and have been UNAVAILABLE 10 MINUTES AGO 
+      # -- which are on a SCHEDULABLE node that was CREATED AT LEAST 10 MINUTES AGO
+    - expr: |
+        (
+          (
+            label_replace(
+              kube_daemonset_status_number_unavailable{cluster_id!="argali|giraffe", namespace=~"giantswarm|kube-system", daemonset=~"aws-node|kiam-server|kiam-agent"},
+              "service", "$1", "workload_name", "(.*)" ) > 0
+            and on (daemonset, node)
+            label_replace(
+              kube_daemonset_status_number_unavailable{cluster_id!="argali|giraffe", namespace=~"giantswarm|kube-system", daemonset=~"aws-node|kiam-server|kiam-agent"} offset 10m,
```

Comment:
if we still ignore china we should add `akita`

Reply:
to all expressions where is this used

Reply:
Oh yeah, you are right.
Although, does that even still make sense? The distinction only works for management-clusters anyway so I'm not sure if it's helpful at all.

Reply:
yeah maybe its useless anyway :p